### ISSUE TITLE
OP_REQUIRES failed at crop_and_resize_op

### ISSUE THREAD

**Issue by JoaoCH** [TannerGilbert/Tensorflow-Object-Detection-API-train-custom-Mask-R-CNN-model#5]
Hi, 

I'm having trouble training with your data. After overcoming problems derived from CUDA / cudNN now I came across the following error:

"OP_REQUIRES failed at crop_and_resize_op". 

Update:
As your images are 800x600 I changed the config file as follows:

image_resizer {
      fixed_shape_resizer {
        height: 600
        width: 800
      }

By making this adjustment I can train.

Is this a mistake or is my system or GPU that is causing this error.

best regards,
John

**Comment by TannerGilbert** [comment]
Normally it should work with the default values since that is what the model was pretrained for and it shouldn't be a problem to upscale the images to the right size.

Maybe open an issue on the [models repository](https://github.com/tensorflow/models/issues) as there are more people there that may know the root of the problem.

**Comment by JoaoCH** [comment]
> Normally it should work with the default values since that is what the model was pretrained for and it shouldn't be a problem to upscale the images to the right size.
> 
Do you think there could be a problem with resizing a rectangular image to a square one.
(resize images with different ratio)

however it is now working.

by the way, Keras must have been updated because I can export inference graph without errors.
